platform/android/java_godot_lib_jni.cpp

```diff
@@ -215,20 +221,13 @@ JNIEXPORT void JNICALL Java_org_godotengine_godot_GodotLib_newcontext(JNIEnv *en
 			os_android->set_context_is_16_bits(!p_32_bits);
 		} else {
 			// GL context recreated because it was lost; restart app to let it reload everything
+			step = -1; // Ensure no further steps are attempted and no further events are sent
 			os_android->main_loop_end();
 			godot_java->restart(env);
-			step = -1; // Ensure no further steps are attempted
 		}
 	}
 }
 
-JNIEXPORT void JNICALL Java_org_godotengine_godot_GodotLib_back(JNIEnv *env, jclass clazz) {
-	if (step == 0)
-		return;
-
-	os_android->main_loop_request_go_back();
-}
-
```

Comment:
This seems to still be used in [`Godot.java`](https://github.com/godotengine/godot/blob/2269c2ebe2cd9eee16bb685cb945d67991d4b27b/platform/android/java/lib/src/org/godotengine/godot/Godot.java#L941).

Reply:
I failed to find the call. Will restore it.

Reply:
Done.